main.go

```diff
@@ -46,58 +27,44 @@ func main() {
 
 func runMain() error {
 	// Check if we're in the root of a Git repo
-	currentWorkingDirpath, err := os.Getwd()
+	currentWorkingDirectory, err := os.Getwd()
 	if err != nil {
-		return stacktrace.Propagate(err, "An error occurred getting the current working directory")
+		return stacktrace.Propagate(err, "An error occurred getting the current working directory.")
 	}
-	gitDirpath := path.Join(currentWorkingDirpath, gitDirname)
-	if _, err := os.Stat(gitDirpath); err != nil {
-		if os.IsNotExist(err); err != nil {
-			return stacktrace.NewError("No Git directory found, meaning this isn't being run from the root of a Git repo")
+	gitrepositoryDirectory := path.Join(currentWorkingDirectory, gitDirname)
```

Comment:
1) same here (and everywhere) in the PR re. "directory" vs "dirpath"
2) needs to be camel-cased